docs/06-ModulesAndHelpers.md

```diff
@@ -264,46 +264,6 @@ In this helper you can replace the parent's methods with your own implementation
 You can also replace the `_before` and `_after` hooks, which might be an option
 when you need to customize starting and stopping of a testing session.
 
-If some of the methods of the parent class should not be used in a child module, you can disable them.
```

Comment:
This part is a bit magical and overcomplicated. I don't think it should be used anymore and probably we will drop this functionality in future releases.